mungegithub/mungers/check-labels_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"testing"
+
+	"github.com/golang/glog"
+	"github.com/google/go-github/github"
+)
+
+var (
+	_                              = fmt.Printf
+	_                              = glog.Errorf
+	repoLabelNames []string        = []string{"team/abc", "team/def", "team/ghi", "label/rep"}
```

Comment:
I had trouble setting it directly because the `github.Label` expects `*strings`. I couldn't find a way to create and initialize a `*string` in a single step. Is there a way to do that? Else I could make it a single array of pairs of strings rather than two separate arrays perhaps.